src/oidcc_token.erl

```diff
@@ -194,6 +196,13 @@
     dpop_nonce => binary()
 }.
 
+-type validate_jwt_opts() ::
+    #{
+        signing_algs => [binary()] | undefined,
```

Comment:
Should that be optional? Without it it can’t ever be valud, no?

Reply:
You need a value for either `signing_algs` or both `encryption_algs`/`encryption_encs`: an encrypted but not signed token would be acceptable though.

I included `undefined` so that a pattern like this is possible:
```elixir
token = ...
{:ok, config} = Oidcc.ProviderConfiguration.Worker.get_provider_configuration(issuer)
Oidcc.Token.validate_jwt(issuer, token, %{
  signing_algs: config.id_token_signing_alg_values_supported,
  encryption_algs: config.id_token_encryption_alg_values_supported,
  encryption_encs: config.id_token_encryption_enc_values_supported
})
```

Reply:
Ah, makes sense